src/2_resources.js

```diff
@@ -297,3 +297,15 @@ resources.event = {
 		"initial_referrer": validator(false, validationTypes.STRING)
 	})
 };
+
+resources.commerceEvent = {
+	destination: config.api_endpoint,
+	endpoint: "/v1/event",
+	method: utils.httpMethod.POST,
+	params: defaults({
+		"event": validator(true, validationTypes.STRING),
+		"metadata": validator(false, validationTypes.OBJECT),
+		"initial_referrer": validator(false, validationTypes.STRING),
+		"commerce_data": validator(true, validationTypes.OBJECT)
```

Comment:
Are we handling this commerce_data param in the api yet? We need to make sure this is how the api wants it